cmd/earthly/main.go

```diff
@@ -362,6 +363,12 @@ func newEarthlyApp(ctx context.Context, console conslogging.ConsoleLogger) *eart
 			Usage:       wrap("Execute in CI mode (implies --use-inline-cache --save-inline-cache --no-output --strict)", "*experimental*"),
 			Destination: &app.ci,
 		},
+		&cli.BoolFlag{
+			Name:        "output",
+			EnvVars:     []string{"EARTHLY_OUTPUT"},
+			Usage:       wrap("Negate --no-output", "*experimental*"),
```

Comment:
Open to suggestions on this description

Reply:
Perhaps something along the lines of `Allow artifacts or images to be output, even when running under --ci mode`?